### ISSUE TITLE
Accessibility Report: Hero Component

### ISSUE THREAD

**Issue by sherakama** [SU-SWS/decanter#319]
Use the AMP tool to scan the [Hero](http://decanter.stanford.edu/section-components.html#kssref-components-hero) component for accessibility issues and document them to this ticket.

**Comment by sherakama** [comment]
AMP Report: https://stanford.levelaccess.net/public/reporting/view_report.php?report_id=283427

Violations: 
* iframe missing title
* change absolute width and height values to percentages

**Comment by sherakama** [comment]
Chrome Report: 93

Violations match AMP report. 

![Screen Shot 2019-03-13 at 9 07 28 PM](https://user-images.githubusercontent.com/550602/54330718-07567280-45d4-11e9-80d6-0a5e7d9e3d4f.png)
